Source/JavaScriptCore/yarr/YarrUnicodeProperties.cpp

```diff
@@ -57,7 +57,7 @@ struct HashTable {
             return -1;
 
         while (true) {
-            if (WTF::equal(key, values[valueIndex].key))
+            if (WTF::equal(key, StringView { values[valueIndex].key }))
```

Comment:
I guess same again, although maybe here we are comparing with a key already known to be ASCII so it’s not so bad to the theoretically wrong thing with non-ASCII.